README.md

```diff
@@ -14,26 +14,9 @@ Subscribe to the following repo to be notified of announcements and updates abou
 
 ## REST API Reference
 
-The following reference pages are available to describe the Azure App Configuration API surface in detail.
-
-**Resources**
-  * [Keys](./docs/REST/keys.md)
-  * [Key-Values](./docs/REST/kv.md)
-  * [Labels](./docs/REST/labels.md)
-  * [Locks](./docs/REST/locks.md)
-  * [Revisions](./docs/REST/revisions.md)
-
-**Protocol**
-  * [Authentication](./docs/REST/authentication/index.md)
-  * [Authorization](./docs/REST/authorization/index.md)
-  * [Consistency Model](./docs/REST/consistency.md)
-  * [Common Headers](./docs/REST/headers.md)
-  * [Throttling](./docs/REST/throttling.md)
-  * [Versioning](./docs/REST/versioning.md)
-
-**Development**
-  * [Fiddler](./docs/Development/fiddler.md)
-  * [Postman](./docs/Development/postman.md)
+Follow links below for the Azure App Configuration REST API reference
+  * [Key-vaule data operation](https://docs.microsoft.com/azure/azure-app-configuration/rest-api)
+  * [Configuration store management](https://docs.microsoft.com/rest/api/appconfiguration/)
```

Comment:
I agree that simple is better. I think you missed a "the" for "Follow the links below", otherwise this looks great

Reply:
Thanks for catching that. Updated.